packages/connect-form/src/components/advanced-options-tabs/ssh-tunnel-tab/proxy-and-ssh-tunnel-tab.spec.tsx

```diff
@@ -0,0 +1,269 @@
+
+    updateConnectionFormFieldSpy = sinon.spy();
+
+    renderWithOptionsAndUrl(
+      {} as ConnectionOptions,
+      connectionStringUrl,
+      updateConnectionFormFieldSpy
+    );
+  });
+
+  it('renders SSH none tab by default', function () {
+    const noneTab = screen.getByTestId('none-tab-content');
+    expect(noneTab).to.exist;
+  });
+
+  it('renders tab when selected', function () {
+    ['none', 'password', 'identity'].forEach(function (type) {
+      const tabButton = screen.getByTestId(`${type}-tab-button`);
+      fireEvent.click(tabButton);
+      const tabContent = screen.getByTestId(`${type}-tab-content`);
+      expect(tabContent, `${type} tab should be selected`).to.exist;
+    });
+  });
+
+  describe('renders tabs correctly', function () {
+    let connectionStringUrl: ConnectionStringUrl;
+    beforeEach(function () {
+      updateConnectionFormFieldSpy = sinon.spy();
+      connectionStringUrl = new ConnectionStringUrl(
+        'mongodb+srv://0ranges:p!neapp1es@localhost/?ssl=true'
+      );
+      cleanup();
+    });
+
+    // eslint-disable-next-line mocha/no-setup-in-describe
+    proxyOptions.forEach((proxyKey) => {
+      it(`renders socks tab when any of socks option is selected - ${proxyKey}`, function () {
+        connectionStringUrl.searchParams.set(proxyKey, 'random-value');
+        renderWithOptionsAndUrl(
+          {} as ConnectionOptions,
+          connectionStringUrl,
+          updateConnectionFormFieldSpy
+        );
+        expect(screen.getByTestId('socks-tab-content')).to.exist;
+      });
+    });
+
+    // eslint-disable-next-line mocha/no-setup-in-describe
+    ['identityKeyPassphrase', 'identityKeyFile'].forEach((identityOption) => {
+      it(`renders identity tab when any of identity option is selected - ${identityOption}`, function () {
+        const connectionOptions: ConnectionOptions = {
+          sshTunnel: {
+            host: '',
+            port: 0,
+            username: '',
+            [identityOption]: 'some-value',
+          },
+          connectionString: connectionStringUrl.toString(),
+        };
+        renderWithOptionsAndUrl(
+          connectionOptions,
+          connectionStringUrl,
+          updateConnectionFormFieldSpy
+        );
+        expect(screen.getByTestId('identity-tab-content')).to.exist;
+      });
+    });
+
+    it(`renders password tab when any of password option is selected`, function () {
+      const connectionOptions: ConnectionOptions = {
+        sshTunnel: {
+          host: '',
+          port: 0,
+          username: '',
+          password: '***',
+        },
+        connectionString: connectionStringUrl.toString(),
+      };
+      renderWithOptionsAndUrl(
+        connectionOptions,
+        connectionStringUrl,
+        updateConnectionFormFieldSpy
+      );
+      expect(screen.getByTestId('password-tab-content')).to.exist;
+    });
+
+    it(`renders none tab when sshTunnel and proxy option is not defined `, function () {
```

Comment:
I’m surprised the linter doesn’t say anything about the backticks, usually it complains for me when I use template strings without interpolation